src/components/TransactionDetails/index.tsx

```diff
@@ -31,23 +51,25 @@ type TransactionDetailsAmountItemProps = Omit<
   ComponentProps<typeof InlineTokenBalance>,
   "tokenAmount"
 > &
-  Pick<TransactionDetailsItemProps, "label"> & { tokenAmount?: string }
+  Omit<TransactionDetailsItemProps, "value"> & {
+    tokenAmount?: string
+  }
 
 export const TransactionDetailsAmountItem: FC<
   TransactionDetailsAmountItemProps
-> = ({ label, tokenAmount, ...restProps }) => {
-  const tokenBalanceTextColor = useColorModeValue("gray.700", "gray.300")
-
+> = ({ label, tokenAmount, isTotal, isSubTotal, ...restProps }) => {
   return (
-    <TransactionDetailsItem label={label}>
+    <TransactionDetailsItem
+      label={label}
+      isTotal={isTotal}
+      isSubTotal={isSubTotal}
+    >
       <Skeleton isLoaded={!!tokenAmount}>
-        <BodySm color={tokenBalanceTextColor}>
-          <InlineTokenBalance
-            withSymbol
-            tokenAmount={tokenAmount || "0"}
-            {...restProps}
-          />
-        </BodySm>
+        <InlineTokenBalance
+          withSymbol
+          tokenAmount={tokenAmount || "0"}
+          {...restProps}
+        />
```

Comment:
Based on the above:

```suggestion
        <InlineTokenBalance
          withSymbol
          tokenAmount={tokenAmount || "0"}
          color={isTotal ? "hsl(182, 100%, 70%)" : "white"}
          fontWeight={isTotal || isSubTotal ? "bold" : "normal"}
          {...restProps}
        />
```

Remember to test it before commiting

Reply:
I don't get it. What is the purpose of this change? Visual changes are provided by the parent node.